config.go

```diff
@@ -184,6 +185,9 @@ func (config *Configuration) defaultSettings() {
 
 func (config *Configuration) expandEnv() {
 	config.AURURL = os.ExpandEnv(config.AURURL)
+	if strings.HasPrefix(config.BuildDir, "~"+string(os.PathSeparator)) {
```

Comment:
this looks weird, but `os.PathSeparator` is a rune not a character, so it needs to be converted/cast to a string. using `filepath.Join()` is also an option here, but because it would need to be written as `filepath.Join("~", string(os.PathSeparator))` I decided there were no benefits over `+`.